src/pages/[platform]/build-a-backend/auth/concepts/external-identity-providers/index.mdx

```diff
@@ -387,6 +387,72 @@ await signInWithRedirect({
 });
 ```
 
+### Redirect URLs
```

Comment:
will this section as be available for RN?

Reply:
Yes, React Native fliter has a similar section, yes.